src/ontology/uberon-edit.obo

```diff
@@ -224175,6 +224175,25 @@ relationship: has_part CL:0000067 ! ciliated epithelial cell
 relationship: part_of UBERON:0011148 ! submucosal gland
 property_value: dcterms-date "2023-05-02T08:09:14Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600015
+name: liver right posterior lobe
+def: "The posterior sector of the right lobe of the liver." [PMID:30992874]
+synonym: "liver right lateral lobe" EXACT [PMID:24453062]
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-6443-9376
+relationship: part_of UBERON:0001114 ! right lobe of liver
+property_value: dcterms-date "2023-05-15T10:46:34Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600016
+name: liver right anterior lobe
+def: "The anterior sector of the right lobe of the liver." [PMID:30992874]
```

Comment:
Regarding text def for both: sector --> segment

Reply:
@aleixpuigb, thank you for clarifying. 

The labels and synonyms include the word "lobe", but the text definition of these classes are describing sectors, not lobes. A similar comment related to brain regions: https://github.com/obophenotype/uberon/issues/2832#issuecomment-1520538497

If the class is meant to represent the sectors, consider renaming the label and synonyms.

Also consider adding synonyms that substitute "liver" with "hepatic" as included in the dbxrefs, e.g., "right hepatic anterior sector" PMID:[30487943](https://pubmed.ncbi.nlm.nih.gov/30487943)